src/librustdoc/config.rs

```diff
@@ -224,6 +228,14 @@ impl Options {
             for &name in passes::DEFAULT_PRIVATE_PASSES {
                 println!("{:>20}", name);
             }
+            println!("\nPasses run with `--show-coverage`:");
```

Comment:
I wonder: is it debug?

Reply:
Are you asking whether this should use `debug!()` instead of `println!()`? All the other prints in this function use `println` so i just copied those.

...though, it's worth considering whether i should only print this on nightly, since the flag is unstable... :thinking: